src/websrc/json/en.json

```diff
@@ -26,10 +26,10 @@
         "kb": "KiB",
         "efti": "internal",
         "efte": "external",
-        "ens": "NV storage",
+        "ens": "Storage",
```

Comment:
Let's leave storage type

Reply:
As u wish, given that there is only one storage, there is no point in calling it an additional semantic load; the presence of the device already indicates that the user is aware of its specifications :+1:

Reply:
But we also have file system that is also "storage" so we need to separate the meanings - File system and Non-Volatile Storage. So that a person who sees the device for the first time, but is familiar with the terminology, would also understand it completely.